src/liblx/common/liblx-config.h

```diff
@@ -62,12 +62,12 @@
  * <tt>libsbml-config-win.h</tt> are all installed by @c "make install".
  */
 
-#ifndef LIBSBXML_CONFIG_H
-#define LIBSBXML_CONFIG_H 1
```

Comment:
Where are these SBXML names coming from?

Reply:
That was Tim originally called the library